### ISSUE TITLE
Rename `version` field to make more clear that this is for the USS version

### ISSUE THREAD

**Issue by ldeboer** [UniversalScheduleStandard/UniversalScheduleStandard#3]
### **The issue:** 
Currently, there is a field on the root universalScheduleStandard object named "version". To most people reading through this object (including myself), I think the version field would be interpreted as containing a version name or number for the production or breakdown data, rather than a place for indicating the current version of the USS standard being used. 

### **Proposed Solution:** 
I would recommend one of the following solutions:

1) Rename the "version" field to `ussVersion` or `formatVersion` or something similar to this to make clear that it should not be used to store project or breakdown version data. 

2) Move the "version" field outside of the universalScheduleStandard object and to the very top of the JSON object, such as this:

```
{
  "version" : "1.0.2" 
  "universalScheduleStandard": {
    "id": string | UUID value | required,
    "author": string | name of individual creator | can be null,
    "company": string | name of the company for which the schedule was created | can be null,
    [ .... ]
  }
}
```

Personally, I think I'm leaning toward option 2. But interested to hear what others think! 

Thanks!
Luke @ SetHero ([luke@setheroapp.com](mailto:luke@setheroapp.com))

**Comment by modemmute** [comment]
Renaming `version` to `ussVersion` makes sense and would be a simple and potentially helpful change to the standard. I can definitely see it decreasing the possibility of misusing the key when implemented. 

Will leave this issue open for a while to allow others to comment.

**Comment by UniversalScheduleStandard** [comment]
This suggestion was accepted and is included in v0.9.1
